cg/models/demultiplex/run_parameters.py

```diff
@@ -315,8 +315,10 @@ def get_flow_cell_model(self) -> str:
         for consumable_info in consumable_infos:
             type_element: Element | None = consumable_info.find("Type")
             if type_element is not None and type_element.text == "FlowCell":
-                name_element: Element | None = consumable_info.find("Name") or consumable_info.find(
-                    "Mode"
-                )
+                name_element: Element | None = consumable_info.find("Name")
                 if name_element is not None:
                     return name_element.text
+                else:
+                    mode_element: Element | None = consumable_info.find("Mode")
+                    if mode_element is not None:
```

Comment:
Is this not clearer?
```suggestion
                    if mode_element:
```

Reply:
I will make another PR to do this properly, right now this is a fix to get it working!